lib/js/components/Outline/OutlineDivider/OutlineDivider.stories.ts

```diff
@@ -0,0 +1,48 @@
+import OutlineDivider from './OutlineDivider.vue';
+import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue3';
+import { DIVIDER_PROMINENCES, DIVIDER_SIZES } from '../../Divider';
+
+export default {
+	title: 'Components/Outline/OutlineDivider',
+	component: OutlineDivider,
+} as Meta<typeof OutlineDivider>;
+
+const StoryTemplate: StoryFn<typeof OutlineDivider> = (args) => ({
+	components: { OutlineDivider },
+	setup() {
+		return { ...args };
+	},
+	data() {
+		return {
+			DIVIDER_SIZES: Object.freeze(DIVIDER_SIZES),
+		};
+	},
+	template: `<outline-divider :prominence="prominence" :size="DIVIDER_SIZES[size]" />`,
+});
+
+export const Interactive = StoryTemplate.bind({});
+
+const args = {
+	defaultSlot: null,
+} as Args;
+
+const argTypes = {
+	prominence: {
+		control: { type: 'select', options: Object.keys(DIVIDER_PROMINENCES) },
+		defaultValue: 'DEFAULT',
```

Comment:
why not const?

Reply:
Because we use key names here not values.

Reply:
but in component props we use default as value and in validation we use `.values` to compare 
```
prominence: {
			type: String,
			default: DIVIDER_PROMINENCES.DEFAULT,
			validator: (value: string) => Object.values(DIVIDER_PROMINENCES).includes(value),
		},
```
and in Divider.stories.ts we have `defaultValue: DIVIDER_PROMINENCES.DEFAULT,`
Are you sure all are correct?

Reply:
Look in the template. We take key and choose correct value from const there. I didn't figure it out I copied this from other stories.

Reply:
but it makes no sense. If someone changes the key, this string value in this storybook will be fucked up